lib/cocoapods/sandbox.rb

```diff
@@ -34,6 +34,14 @@ module Pod
   #     +-- Manifest.lock
   #     |
   #     +-- Pods.xcodeproj
+  #  (if installation option 'generate_multiple_pod_projects' is enabled)
+  #     |
+  #     +-- PodTarget1.xcodeproj
```

Comment:
I think this might be to be indented as it appears they are siblings now.

Reply:
I think this is a representation of the sandbox dir in which each of the pod targets are siblings in the filesys. @segiddins and I discussed moving them into a subdirectory as a follow up to keep the scope of this PR a bit smaller since it would involve a bit of refactoring.

Reply:
ah yeah sg!